kolibri/plugins/coach/serializers.py

```diff
@@ -68,12 +67,11 @@ def get_last_active(self, target_user):
 class ContentReportSerializer(serializers.ModelSerializer):
     progress = serializers.SerializerMethodField()
     last_active = serializers.SerializerMethodField()
-    parent = serializers.SerializerMethodField()
 
     class Meta:
         model = ContentNode
         fields = (
-            'pk', 'content_id', 'title', 'progress', 'kind', 'last_active', 'parent',
+            'pk', 'content_id', 'title', 'progress', 'kind', 'last_active',
```

Comment:
Doesn't get used anywhere in the frontend code, I grepped and grepped, and found only a vuex setter using it, but no reference to that set data. It breaks nothing!

The breadcrumbs are constructed using the `get_ancestor` data on the ContentSummarySerializer.